poup.css

```diff
@@ -0,0 +1,233 @@
+.popup_window {
+  position: fixed;
+  width: 100%;
+  height: 100vh;
+  display: none;
+  top: 1%;
+  bottom: 0;
+  overflow: auto;
+}
+
+.popup_skills_mob li {
+  font-size: 12px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.36px;
+  background-color: #ebecf0;
+  color: #344563;
+  padding: 10px 10px;
+  border-radius: 5px;
+  margin-right: 1%;
+}
+
+.popup {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 13px;
+  justify-content: center;
+  width: 80%;
+  padding: 75px 0;
+  margin-left: 8%;
+  background-color: #fff;
+}
+
+.popup .mob_snap {
+  width: 90%;
+  text-align: center;
+  align-items: center;
+
+  /* display: flex;
+   height: 220px;
+  padding: 8px 8px 172px 263px;
+  justify-content: flex-end;
+  align-items: center;
+  border-radius: 8px; */
+}
+
+.popup_btn_src img {
+  color: #fff;
+  width: 18px;
+  height: 20px;
+  flex-shrink: 0;
+}
+
+.popup_btn_live img {
+  color: #fff;
+  width: 24px;
+  height: 24px;
+}
+
+.popup .desktop_snap {
+  display: none;
+}
+
+.popup_heading h3 {
+  display: flex;
+  flex-direction: column;
+  color: #172b4d;
+  font-size: 32px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 44px;
+}
+
+.popup_heading {
+  width: 60vw;
+  margin-top: 20px;
+}
+
+.popup_skills_desktop {
+  display: none;
+}
+
+.popup_skills_mob {
+  display: flex;
+  margin: 15px 0;
+  color: #3a4a42;
+  font-size: 12px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.36px;
+}
+
+.popup-page {
```

Comment:
- [ ] Kindly adhere to the project requirement because as I can see you are not using the `width` and `margin` properties correctly. which makes your design `inconsistent` and looks unprofessional. Please have a look at the below-attached image and table.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![image](https://github.com/Cheptoo21/Portfolio-setup-and-mobile-first/assets/66476329/e17d5a4c-5254-477d-a21c-1900f3dd4838)

</td>
<td>

![Screenshot from 2023-07-02 00-03-44](https://github.com/Cheptoo21/Portfolio-setup-and-mobile-first/assets/66476329/f810bf7d-9998-45e3-b238-169af2ca9f6a)

</td>
</tr>
</table>